### ISSUE TITLE
Flutter_ data dependence on http

### ISSUE THREAD

**Issue by ouzari** [flutterdata/flutter_data#250]
When i try to get the flutter_data package in my project , I get the following message:
```
Because myApp depends on flutter_data ^1.5.6 which depends on http ^0.13.3, http ^0.13.3 is required.
So, because myApp depends on http ^1.1.0, version solving failed.
exit code 1
```

So could you please update the http version used in flutter_data.

**Comment by oonxt** [comment]
same issue.

Because google_fonts >=4.0.5 depends on http ^1.0.0 and flutter_data >=0.8.0-rc1 depends on http ^0.13.3, google_fonts >=4.0.5 is incompatible with flutter_data >=0.8.0-rc1.

**Comment by jantech** [comment]
Workaround: try using 

dependency_overrides:
  http: ^1.0.0

will solve the error.

**Comment by frank06** [comment]
Fixed in 1.5.15
